source/service_test.go

```diff
@@ -1541,9 +1541,9 @@ func TestNodePortServices(t *testing.T) {
 					},
 				},
 			}},
-			[]string{"pod-0"},
-			[]int{1},
-			[]v1.PodPhase{v1.PodRunning},
+			[]string{"pod-0", "pod-1"},
+			[]int{1, 1},
+			[]v1.PodPhase{v1.PodRunning, v1.PodRunning},
```

Comment:
I think we should add a new test case instead of modifying an existing test case. Something like `annotated NodePort services with multiple pods on a single node`.

Reply:
Done